app/api/protocol/vaults/fetch-holders.ts

```diff
@@ -0,0 +1,26 @@
+import type { ApolloClient } from '@apollo/client'
+import { getDeposits } from '../queries/get-deposit.query'
+
+/**
+ * (!) This is hacky and naive approach to calcaute total holders.
+ * TODO: Consider to replace this logic later.
+ */
+export async function fetchHolders(client: ApolloClient<any>): Promise<string[]> {
```

Comment:
Great stuff, but really too advance for us now. We have not enough people to load it in real time. 
Can create test for this function, so it not will break over time and just hardcode 99+ in landing for now